zokrates_field/Cargo.toml

```diff
@@ -10,10 +10,15 @@ serde_derive = "1.0"
 lazy_static = "1.4"
 bincode = "0.8.0"
 serde_json = "1.0"
+bellman_ce = { version = "^0.3", default-features = false }
+algebra-core = { git = "https://github.com/scipr-lab/zexe.git", default-features = false }
 sha2 = "0.8.0"
 num-traits = { version = "0.2", default-features = false }
 num-integer = { version = "0.1", default-features = false }
-bellman_ce = { version = "^0.3", default-features = false }
+
+[dependencies.algebra]
+git = "https://github.com/scipr-lab/zexe.git"
```

Comment:
Do we have to rely on github?

Reply:
Yeah it seems to not be published as a crate

Reply:
Could we target a specific commit?  `38b6f6f` was used to release some sub-crates https://github.com/scipr-lab/zexe/releases

Reply:
Will open issue for this